packages/browserify/package.json

```diff
@@ -44,7 +44,7 @@
   },
   "devDependencies": {
     "@babel/code-frame": "7.25.7",
-    "browserify": "17.0.0",
+    "browserify": "17.0.1",
```

Comment:
Wait, why does this package depend on itself?

Reply:
`@lavamoat/browserify` is depending on `browserify` - `browserify` isn't depending on ``browserify`?

(BTW I always found it unfortunate with calling the workspaces `node`/`browserify`/`webpack` because of potential confusion like this.. Vs anything less ambiguous like `node-runner`/`browserify-plugin`/`webpack-plugin`)

Reply:
ok, I shouldn't have been doing reviews while ill afterall :D